### DISCUSSION TITLE
Can't place blocks in tshock

### DISCUSSION THREAD

**Discussion by PiesElphur** [Pryaxis/TShock#2084] [Q&A]
Every time I try to place a block, it instantly disapears. I do have superadmin. I can destroy blocks and block swap blocks. I can place walls and furniture. I cannot place blocks. "DisableBuild" is false in the config.json file. Any suggestions? I stood up my own server for tshock with the latest tshock version. Everything appears to be working except this.

**Comment by hakusaro** [comment]
Can you upload a video of it happening?

GitHub <notifications@github.com> wrote:
“

Every time I try to place a block, it instantly disapears. I do have superadmin. I can destroy blocks and block swap blocks. I can place walls and furniture. I cannot place blocks. "DisableBuild" is false in the config.json file. Any suggestions?
